src/solana/SolanaEngine.ts

```diff
@@ -747,10 +747,14 @@ export class SolanaEngine extends CurrencyEngine<
       solTx.add(memoOpts)
     }
 
-    const otherParams: JsonObject = {
-      unsignedSerializedSolTx: solTx.serialize({
+    const unsignedSerializedSolTx = base64.stringify(
+      solTx.serialize({
         requireAllSignatures: false
       })
+    )
+
+    const otherParams: JsonObject = {
```

Comment:
Does Solana use a cleaner for `otherParams`? That cleaner should be responsible for serializing / unserializing the payload using `asBase64`. Any time we have raw `base64.parse` or `base64.serialize` calls, that's a big red flag that suggests we aren't cleaning stuff properly.